### PR TITLE
Bug 1797643: Set shorter names when generating pipeline on import flows

### PR DESCRIPTION
**Pull request by rohitkrai03**
Fixes - https://issues.redhat.com/browse/ODC-2761

This PR 
- Shortens the pipeline name that gets generated using pipeline template in import flows.
- Adds unit tests and fixes old ones.

### CODE REVIEW

File: frontend/packages/dev-console/src/components/import/pipeline/pipeline-template-utils.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const createPipelineForImportFlow = async (formData: GitImportFormData) =
   const template = _.cloneDeep(pipeline.template);
 
   template.metadata = {
-    name: `${name}-${template.metadata.name}`,
+    name: `${name}`,
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/components/import/pipeline/pipeline-template-utils.ts]: @rohitkrai03 I assume you tested the scenario about an existing Pipeline with this name? This hopefully errors out like any other resource we create when names duplicate.

Comment (rohitkrai03) [reply frontend/packages/dev-console/src/components/import/pipeline/pipeline-template-utils.ts]: @andrewballantyne Yes, we get that for every kubernetes resource.

Comment (andrewballantyne) [reply frontend/packages/dev-console/src/components/import/pipeline/pipeline-template-utils.ts]: I assumed as much - just wanted to make sure... it was an explicit agreement with PM.

**Pull request merged**